### ISSUE TITLE
Attach photo to transaction

### ISSUE THREAD

**Issue by mvarnagiris** [mvarnagiris/financius#29]

**Comment by marcovrv** [comment]
It would be nice to use this to scan receipts with OCR. There are some OSS libraries that might help. i.e. http://gaut.am/making-an-ocr-android-app-using-tesseract/

**Comment by mvarnagiris** [comment]
Yup that's the plan. Thanks for posting the link to the library!

**Comment by ghost** [comment]
This is the only thing I miss in this app. Any progress on this?

**Comment by auzair92** [comment]
This is the only thing I miss in this app. It is absolutely perfect according to my needs otherwise. I hope this feature is added soon!
